README.md

```diff
@@ -230,7 +230,7 @@ Additional examples can be found in the [examples](https://github.com/embedded-g
 
 ## Minimum supported Rust version
 
-The minimum supported Rust version for embedded-graphics is `1.56.1` (edition 2021) or greater.
+The minimum supported Rust version for embedded-graphics is `1.57` (edition 2021) or greater.
```

Comment:
The chosen version is no longer actually edition 2021. That said, should we still mention that this chosen version is "edition 2021 + a bit"?

```suggestion
The minimum supported Rust version for embedded-graphics is `1.57` or greater.
```

Don't forget the other 3 places too.

Reply:
Ok, updated to remove mentions of the edition. IMO it'd be a little confusing to mention the edition since 1.57 still supports the 2021 edition, and it doesn't matter to consumers of the library anyway, since they can remain on a previous edition as long as they have 1.57.

Reply:
That makes sense yeah :)